webservice/src/test/java/com/thoughtworks/aceleradora/repository/StageRepositoryTest.java

```diff
@@ -0,0 +1,40 @@
+package com.thoughtworks.aceleradora.repository;
+
+import com.thoughtworks.aceleradora.domain.Question;
+import com.thoughtworks.aceleradora.domain.Stage;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StageRepositoryTest {
```

Comment:
Acredito que essa classe não é necessária. Porque ela está fazendo testes redundantes. Vocês estão testando um mock ao invés de realmente testar o repositório. Se julgarem realmente necessário testar o repository, usem esse link: [Acesse aqui](https://stackoverflow.com/questions/23435937/how-to-test-spring-data-repositories)

Porem acredito que não é necessário, vendo que o repository é do spring data, e o mesmo já foi testado de diversas formas sua funcionalidade.